src/core/etl/src/Flow/ETL/Loader/BatchingLoader.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Flow\ETL\Loader;
+
+interface BatchingLoader
+{
+    /**
+     * @return int<1, max>
+     */
```

Comment:
Lets describe that method purpose, something that will say the purpose of this interface is to set default batch size and that it's only applied when batch size is not explicitly set through `DataFrame::batchSize()`